pkgs/development/python-modules/f90nml/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchFromGitHub, setuptools-scm }:
+
+buildPythonPackage rec {
+  pname = "f90nml";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "marshallward";
+    repo = pname;
+    rev = "v" + version;
+    sha256 = "sha256-nSpVBAS2VvXIQwYK/qVVzEc13bicAQ+ScXpO4Rn2O+8=";
+  };
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  doCheck = true;
```

```suggestion
```
that is the default which we don't need to repeat.
I was more thinking we need something like pytest. Right now it looks like we are not executing any tests.
We should probably do something like:

```suggestion
  checkPhase = ''
    ${python.interpreter} setup.py check
  '';

```

---

Test are now properly executed, thanks.